apps/src/sites/studio/pages/foorm/forms/editorHelpers.js

```diff
@@ -0,0 +1,52 @@
+import {getStore} from '@cdo/apps/redux';
+import initializeCodeMirror from '@cdo/apps/code-studio/initializeCodeMirror';
+import {
+  setQuestions,
+  setHasJSONError
+} from '@cdo/apps/code-studio/pd/foorm/editor/foormEditorRedux';
+import _ from 'lodash';
+
+let codeMirror;
+
+function populateCodeMirror() {
+  const codeMirrorArea = document.getElementsByTagName('textarea')[0];
+
+  const updateQuestions = questions => {
```

Comment:
it's not good practice to mix conventions in a single module. you're primarily using `function` syntax in here (which as i said i'm in favor of doing everywhere), so this and `confirmNoUnsavedChanges` look out of place.

Reply:
Yeah that was a straight copy/paste, fixed!